container-images/assign-class-label/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
+
+WORKDIR /app/
+
+COPY requirements.txt /app/
```

Since you've just performed `WORKDIR /app` in the previous step, you can just copy into the current directory instead:

```suggestion
COPY requirements.txt ./
```